cmake/CustomTargets.cmake

```diff
@@ -11,15 +11,23 @@ find_program(FORD_EXE ford
   DOC "path to the ford executable (required to generate the documentation)"
   )
 
-# copy the FORD project-file into the build directory
-configure_file(project-file.md.in project-file.md)
+# copy the FORD project-file into the build directory   UPDATE THIS!!!
+set (FORD_PROJECT_FILE "${CMAKE_BINARY_DIR}/DBCSR.md")
+configure_file(DBCSR.md "${FORD_PROJECT_FILE}")
```

Comment:
from [the cmake documentation](https://cmake.org/cmake/help/latest/command/configure_file.html):
```
<output>
Path to the output file or directory. A relative path is treated with respect to the value of
CMAKE_CURRENT_BINARY_DIR. If the path names an existing directory the output file
is placed in that directory with the same file name as the input file.
```
... and the reason why I hardcoded `project-file.md`.

Reply:
Aha, I see... I understand your point, though I'm not the biggest fan of these implicit paths CMake adds. I find it makes it harder to reason about what CMake is doing. 

I didn't want to hard-code the project file's name because it is re-used a few lines below, in the ford invocation.

Reply:
> Aha, I see... I understand your point, though I'm not the biggest fan of these implicit paths CMake adds. I find it makes it harder to reason about what CMake is doing.

yeah, CMake and its idiosyncrasies. I don't mind one way or another, but usually you can assume that CMake does everything to keep the source directory untouched and does not make any assumptions about build-, source- and current-working-directory relationship. From there most behavior becomes obvious, like that a command name in `add_test(... command ...)` will be resolved to a full path if it matches an `add_executable(...)` (which is helpful when building on platforms requiring suffixes for executables).

> I didn't want to hard-code the project file's name because it is re-used a few lines below, in the ford invocation.

Sure, as soon as it's not a simple name it fully makes sense to factor it out.